### DISCUSSION TITLE
Feature: Buy Game that you have claimed

### DISCUSSION THREAD

**Discussion by FrancescoCardone** [ChristopherKlay/StadiaEnhanced#235] [Ideas & Requests]
Hi there :)
It's possible to add this feature? Because for now i have to modify the html page in order to buy a game that i have claimed with the pro

**Comment by ChristopherKlay** [comment]
It sadly isn't, due to each purchase option (i.e. a games base price, each deal and such) having a different purchase ID per title. There's also issues when injected purchase options are not used quickly sometimes, where Stadia just throws out errors when trying to use them.
